### PR TITLE
remove external css and aplly tailwind css

### PR DESCRIPTION
**Pull request by Pks20**

### CODE REVIEW

File: user-posts/components/FooterComponent.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+defineProps<{
+  currentPage: number;
+  totalPages: number;
+  goToPage: (page: number) => void;
```
Comment (10shubham01) [inline user-posts/components/FooterComponent.vue]: check this out 
https://vuejs.org/guide/components/events

File: user-posts/components/FormModal.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  <div
+    v-if="isVisible"
+    class="fixed inset-0 flex items-center justify-center bg-black bg-opacity-50 z-50"
+  >
+    <div class="bg-white rounded-lg shadow-xl p-6 w-full max-w-lg">
+      <h2 class="text-xl font-bold mb-4">
+        {{ isUpdate ? "Update Post" : "Create New Post" }}
+      </h2>
+
+      <form @submit.prevent="handleSubmit">
```
Comment (10shubham01) [inline user-posts/components/FormModal.vue]: use https://ui.nuxt.com/components/form

File: user-posts/components/FormModal.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
```
Comment (10shubham01) [inline user-posts/components/FormModal.vue:1]: https://ui.nuxt.com/components/modal

File: user-posts/pages/index.vue [hunk 4]
```diff
@@ -46,70 +49,105 @@ if (totalData.value) {
...
+      .then((json) => posts.value.push(json));
+  }
+};
+
+const handleEdit = (post: Post) => {
+  isModalVisible.value = true;
+  isUpdate.value = true;
+  selectedPost.value = { ...post };
+};
+
```
Comment (10shubham01) [inline user-posts/pages/index.vue]: Refactor this ... use https://ui.nuxt.com/components/notification to show toast for error or successful CRUD

**Review by 10shubham01 — CHANGES_REQUESTED** [review]

**Review by 10shubham01 — APPROVED** [review]

**Review by archit-credilio — APPROVED** [review]

**Pull request merged**